### DISCUSSION TITLE
Is Protecc alive and well?

### DISCUSSION THREAD

**Discussion by ezxpro** [FireCubeStudios/Protecc#75] [Q&A]
I see the last commit is from October 2022. 
I wish I could help with development, but I don't know any WPF/UWP, so I  really can't help atm.

Cool project, it would be sad to see it go.

**Comment by FireCubeStudios** [comment]
> I see the last commit is from October 2022. 
> I wish I could help with development, but I don't know any WPF/UWP, so I  really can't help atm.
> 
> Cool project, it would be sad to see it go.

Yes it's alive. I usually update apps every year and plan a big one this year but some time needed as I have exams

**Reply by ezxpro** [reply]
Pretty cool, it's a great project.
